apps/spruce/src/constants/requesters.ts

```diff
@@ -0,0 +1,43 @@
+import { StringMap } from "types/utils";
+
+const adhocRequester = "ad_hoc";
+const commitQueueRequester = "merge_test";
+const githubMergeRequester = "github_merge_request";
+const githubPRRequester = "github_pull_request";
+const gitTagRequester = "git_tag_request";
+const gitterRequester = "gitter_request";
+const patchRequester = "patch_request";
+const triggerRequester = "trigger_request";
```

Comment:
I know these values were already set, but thoughts on representing this with an enum? Then the types below can be `PartialRecord` `Record<Requester, string>`